build/parseSpec.c

```diff
@@ -1129,3 +1155,29 @@ rpmSpec rpmSpecParse(const char *specFile, rpmSpecFlags flags,
 {
     return parseSpec(specFile, flags, buildRoot, 0);
 }
+
+rpmRC parseGeneratedSpecs(rpmSpec spec)
+{
+    ARGV_t argv = NULL;
+    int argc = 0;
+    int i;
+    rpmRC rc = RPMRC_OK;
+
+    char * specPattern = rpmGenPath("%{specpartsdir}", NULL, "*.specpart");
+    if (rpmGlob(specPattern, &argc, &argv) == 0) {
```

Comment:
One more thing I realized earlier today but ended up on a long detour... there's a subtle but important thing silently done by rpmGlob() here: it sorts the returned files. Without which, users of this feature may get a pretty weird spec salad of the day. I'd add a comment here because that's a not-so-known feature of rpmGlob(), and also document the order (sorted according to C locale) because it's something users will need to care about.